src/uipath/_cli/_evals/_models/_evaluation_set.py

```diff
@@ -78,6 +78,14 @@ class ModelSettings(BaseModel):
     max_tokens: int | None = Field(default=None, alias="maxTokens")
 
 
+class EvaluationSetModelSettings(BaseModel):
```

Comment:
Should we reuse `ModelSettings` instead? `EvaluationSetModelSettings` seems to be a subset of `ModelSettings`, so it should be forward-compatible.